src/createServer.js

```diff
@@ -1,3 +1,45 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/');
+const { writeErrorMessage } = require('./writeErrorMessage');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const textToConvert = req.url.split('?')[0].slice(1);
+    const query = req.url.split('?')[1];
+    const params = new URLSearchParams(query);
+    const toCase = params.get('toCase');
+    const errors = writeErrorMessage(textToConvert, toCase);
+
+    if (errors.length > 0) {
+      res.statusCode = 400;
+      res.statusMessage = 'Bad request';
+
+      const responseError = {
+        errors,
+      };
+
+      res.end(JSON.stringify(responseError));
```

Comment:
Since you send response in 2 places you can create a separate function `sendResponse(res, data, statusCode)` and move this logic there